async-book/src/execution/executor.md

```diff
@@ -0,0 +1,160 @@
+# Applied: Build an Executor
+
+For this one, we're going to have to include the `futures` crate in order to
+get the `FutureObj` type, which is a dynamically-dispatched `Future`, similar
```

Comment:
Should update this now that `Future` is object-safe.

Reply:
`Pin<Box<dyn Future<Output = ()> + Send>>` is too long compared to `FutureObj<()>`. We need an alias ;)